packages/allure-playwright/src/helpers.ts

```diff
@@ -29,118 +22,82 @@ export class allure {
     });
   }
 
-  static link(link: Link) {
+  static link(url: string, name?: string, type?: string) {
     this.addMetadataAttachment({
-      links: Array.isArray(link) ? link : [link],
+      links: [{ url, name, type }],
     });
   }
 
   static id(id: string) {
-    this.label({
-      value: id,
-      name: LabelName.AS_ID,
-    });
+    this.label(LabelName.AS_ID, id);
   }
 
   static epic(epic: string) {
-    this.label({
-      name: LabelName.EPIC,
-      value: epic,
-    });
+    this.label(LabelName.EPIC, epic);
   }
 
   static feature(epic: string) {
-    this.label({
-      name: LabelName.FEATURE,
-      value: epic,
-    });
+    this.label(LabelName.FEATURE, epic);
   }
 
   static story(story: string): void {
-    this.label({
-      name: LabelName.STORY,
-      value: story,
-    });
+    this.label(LabelName.STORY, story);
   }
 
   static suite(name: string): void {
-    this.label({
-      name: LabelName.SUITE,
-      value: name,
-    });
+    this.label(LabelName.SUITE, name);
   }
 
   static parentSuite(name: string) {
-    this.label({
-      name: LabelName.PARENT_SUITE,
-      value: name,
-    });
+    this.label(LabelName.PARENT_SUITE, name);
   }
 
   static layer(layerName: string) {
-    this.label({
-      name: LabelName.LAYER,
-      value: layerName,
-    });
-  }
-
-  static addParameter(name: string, value: string, options?: ParameterOptions) {
-    this.addMetadataAttachment({
-      parameter: [
-        {
-          name,
-          value,
-          ...options,
-        },
-      ],
-    });
+    this.label(LabelName.LAYER, layerName);
   }
 
   static subSuite(name: string) {
-    this.label({
-      name: LabelName.SUB_SUITE,
-      value: name,
-    });
+    this.label(LabelName.SUB_SUITE, name);
   }
 
   static owner(owner: string) {
-    this.label({
-      name: LabelName.OWNER,
-      value: owner,
-    });
+    this.label(LabelName.OWNER, owner);
   }
 
   static severity(severity: string) {
-    this.label({
-      name: LabelName.SEVERITY,
-      value: severity,
-    });
+    this.label(LabelName.SEVERITY, severity);
   }
 
-  static tag(...tags: string[]) {
-    for (const tag of tags) {
-      this.label({
-        name: LabelName.TAG,
-        value: tag,
-      });
-    }
```

Comment:
Hi! What is the reason for changing adding tags?

Reply:
this function is changing because we want to metadata functions to be the same across the frameworks.

  | playwright | Cucumber | commons | mocha | newman | jasmine
-- | -- | -- | -- | -- | -- | --
tag | `allure.tag(...tags: string[])` | `this.tag(tag: string)` | `tag(tag: string): void;` | `tag(tag: string): void;` | `@allure.label.tag=tagName` |  

Is it really common case to attach multiple tags at once? if so we can add additional helper `allure.tags()` for this use case

Reply:
We use Allure TestOps and use many tags to group and report test runs. Each test can have 4 or more tags.

`allure.tags()` this method would be good, yes.

Reply:
@vovsemenv I'm ok with adding additional bulk helper methods such as `links`, 'labels' and `tags`